### ISSUE TITLE
Change 'allowRowDragAndDrop' from true to false, I expect the drag icon in front of each row to disappear but it's still  visible

### ISSUE THREAD

**Issue by yxrgds** [syncfusion/ej2-angular-ui-components#124]
Change 'allowRowDragAndDrop' from true to false, I expect the drag icon in front of each row to disappear but it's still  visible. There is a example: https://stackblitz.com/edit/angular-8a8g4w?file=app.component.ts. It is designed to act like this?

**Comment by poojapriya-krishnamoorthy** [comment]
Hi Yxrgds,
We are able to reproduce the reported issue. We logged bug report for this. The status can be tracked from below feedback link. 
https://www.syncfusion.com/feedback/17951/provide-dynamic-support-to-enable-disable-drag-and-drop
We will fix this issue and the fix will be included in our upcoming release which is expected to be rolled out on September 22,2020.
Now we have resolved this by work around solution. Please find the below code example.

```
public toolbarClick(args: ClickEventArgs): void {
  if (args.item.id === 'toolbarswitch') {
    this.allowRowDragAndDrop = !this.allowRowDragAndDrop;
    **this.ganttObj.treeGrid.allowRowDragAndDrop = this.allowRowDragAndDrop;**
  }
};
```

Sample - https://stackblitz.com/edit/angular-8a8g4w-9uca1p?file=app.component.ts


Regards,
Pooja K.

**Comment by gopinathmuniraj** [comment]
Hi Yxrgds,

Thanks for your patience.

We are glad to announce that, the reported issue has been fixed in our latest version **v18.2.59**. Please upgrade your Gantt packages to this version.

Release Notes: https://ej2.syncfusion.com/angular/documentation/release-notes/18.2.59/?type=all#gantt

We have prepared a sample with the latest version and you can get the sample from the below link.
Sample: https://stackblitz.com/edit/rowdd-fix?file=app.component.ts

Thanks,
Gopinath M

**Comment by yxrgds** [comment]
> Hi Yxrgds,
> 
> Thanks for your patience.
> 
> We are glad to announce that, the reported issue has been fixed in our latest version **v18.2.59**. Please upgrade your Gantt packages to this version.
> 
> Release Notes: https://ej2.syncfusion.com/angular/documentation/release-notes/18.2.59/?type=all#gantt
> 
> We have prepared a sample with the latest version and you can get the sample from the below link.
> Sample: https://stackblitz.com/edit/rowdd-fix?file=app.component.ts
> 
> Thanks,
> Gopinath M

Hi gopinathmuniraj ,

We have update to the latest version. It did sovle the problem. Thanks.

Hi Yxrgds
